File: CS/WindowsApplication8/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraEditors;

namespace WindowsApplication8
{
    public partial class Form1 : RibbonForm
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            for (int i = 0; i < 10; i++) {
                AppMenuFileLabel f = new AppMenuFileLabel();
                popupControlContainer1.Controls.Add(f);
                f.Dock = DockStyle.Top;
                f.Text = "Document " + i.ToString(); 
                if ((i + 1)% 3 == 0)
                    f.AddSeparator();
            }
        }
    }
}

File: VB/WindowsApplication8/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraEditors

Namespace WindowsApplication8
	Partial Public Class Form1
		Inherits RibbonForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			For i As Integer = 0 To 9
				Dim f As New AppMenuFileLabel()
				popupControlContainer1.Controls.Add(f)
				f.Dock = DockStyle.Top
				f.Text = "Document " & i.ToString()
				If (i + 1) Mod 3 = 0 Then
					f.AddSeparator()
				End If
			Next i
		End Sub
	End Class
End Namespace